### PR TITLE
Readme - add OneOf to pattern list

### PR DESCRIPTION
**Pull request by timabell**

### CODE REVIEW

File: src/Dfc.CourseDirectory.WebV2/README.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ For an intro to MediatR in controllers see:
 * https://jonhilton.net/2016/06/06/simplify-your-controllers-with-the-command-pattern-and-mediatr/
 * https://alexlindgren.com/posts/thin-controllers-using-mediatr-with-aspnet-mvc/
 
+### OnOf
```
Comment (jameslawesynergy) [inline src/Dfc.CourseDirectory.WebV2/README.md:25]: Typo

File: src/Dfc.CourseDirectory.WebV2/README.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ For an intro to MediatR in controllers see:
...
+
+For extra compile-time safety we're making heavy use of the [OneOf](https://www.nuget.org/packages/OneOf/) library / pattern.
```
Comment (jameslawesynergy) [inline src/Dfc.CourseDirectory.WebV2/README.md:27]: In which circumstances are we using int?

**Review by jameslawesynergy — CHANGES_REQUESTED** [review]

File: src/Dfc.CourseDirectory.WebV2/README.md (hunk 2 above)
Comment (timabell) [reply src/Dfc.CourseDirectory.WebV2/README.md:27]: I'm not really sure :grin:

File: src/Dfc.CourseDirectory.WebV2/README.md (hunk 1 above)
Comment (timabell) [reply src/Dfc.CourseDirectory.WebV2/README.md]: oops. fixed.

File: src/Dfc.CourseDirectory.WebV2/README.md (hunk 2 above)
Comment (timabell) [reply src/Dfc.CourseDirectory.WebV2/README.md:27]: @gunndabad maybe you could answer that one?

**Review by jameslawesynergy — APPROVED** [review]

File: src/Dfc.CourseDirectory.WebV2/README.md (hunk 2 above)
Comment (timabell) [reply src/Dfc.CourseDirectory.WebV2/README.md]: discussed on slack

**Pull request merged**